### PR TITLE
Add Support to Encounter Class UI

### PR DESCRIPTION
**Pull request by yashrajbothra**
Signed-off-by: Yash Bothra <yashrajbothra786@gmail.com>

Fixes #

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: interface/forms/newpatient/common.php [hunk 1]
```diff
@@ -491,6 +491,12 @@ function cancelClickedOld() {
                                 ?>
                             </select>
                         </div>
+                        <div class="col-sm-2">
+                            <label for='class' class="oe-text-to-right"><?php echo xlt('Class'); ?>:</label>
+                        </div>
+                        <div class="col-sm">
+                            <?php echo generate_select_list('class_code', '_ActEncounterCode', $viewmode ? $result['class_code'] : 'AMB', ''); ?>
```
Comment (bradymiller) [inline interface/forms/newpatient/common.php:498]: hi @yashrajbothra , Would do this instead:
```
<?php echo generate_select_list('class_code', '_ActEncounterCode', $viewmode ? $result['class_code'] : '', '', ''); ?>
```
(in this case the list itself will enforce the default and that empty 5th parameter will remove the "empty" selection)

**Comment by bradymiller** [comment]
hi @yashrajbothra , Just noted minor issue (see above post). When this is addressed, then will be ready for codebase. thanks! -brady

File: interface/forms/newpatient/common.php (hunk 1 above)
Comment (yashrajbothra) [inline interface/forms/newpatient/common.php:498]:
```suggestion
                            <?php echo generate_select_list('class_code', '_ActEncounterCode', $viewmode ? $result['class_code'] : '', '', ''); ?>

```

**Comment by bradymiller** [comment]
hi @yashrajbothra , Looks good and bringing into codebase. thanks! -brady

**Pull request merged**